pkg/storaged/stratis-details.jsx

```diff
@@ -656,9 +652,8 @@ export const StratisPoolDetails = ({ client, pool }) => {
             <CardBody className="contains-list">
                 <ListingTable emptyCaption={_("No filesystems")}
                               aria-label={_("Filesystems")}
-                              columns={[_("Name"), _("Used for"), _("Size"), _("Actions"), _("Menu")]}
+                              columns={[_("Name"), _("Used for"), _("Size")]}
                               showHeader={false}
-                              variant="compact"
```

Comment:
Hmm are you sure about this? It looks quite bulky now.

Now:
![Screen Shot 2021-11-18 at 15 50 58](https://user-images.githubusercontent.com/14921356/142440026-371d1155-7937-4f02-95c4-0258840b60b8.png)

Before:
![Screen Shot 2021-11-18 at 15 50 48](https://user-images.githubusercontent.com/14921356/142440037-fdaddac1-8c34-45ae-8473-df8b6f8c18da.png)

Reply:
@garrett?